_modules/ash_linux.py

```diff
@@ -51,10 +51,15 @@ def _get_default_kernel():
     return __salt__['cmd.run'](cmd, python_shell=True, output_loglevel='quiet')
 
 
-def _get_boot_mount():
-    """Obtain mount to path that kernel is located in."""
+def _get_boot_path():
+    """Obtain path that kernel is located in."""
     boot_path = _get_default_kernel()
     boot_path = boot_path[:boot_path.rindex('/')] or '/'
```

Comment:
Instead of re-assigning and returning, just:
```suggestion
    return boot_path[:boot_path.rindex('/')] or '/'
```

Reply:
Done.